tests/php/sync/test_class.jetpack-sync-callables.php

```diff
@@ -87,7 +87,7 @@ public function test_sync_callable_whitelist() {
 			'shortcodes'                       => Jetpack_Sync_Functions::get_shortcodes(),
 			'roles'                            => Jetpack_Sync_Functions::roles(),
 			'timezone'                         => Jetpack_Sync_Functions::get_timezone(),
-			'available_jetpack_blocks'         => Jetpack_Gutenberg::get_block_availability(),
+			'available_jetpack_blocks'         => Jetpack_Gutenberg::get_availability(),
```

Comment:
Could `available_jetpack_blocks ` also be renamed to accommodate for the fact that this array contains plugins too?

Reply:
I didn't dare to change this because it's a the name of a synced value, so I was worried it might show up wrong for existing sites that are already using this name. But I have no idea if that is really a problem.

Maybe a @Automattic/jetpack-crew member can advise if it's fine to simply change this name? :smile:

Reply:
As discussed in Slack, not so simple. It would need to be accounted for in the wpcom-side whitelist, and then leave us with another question of whether or not we keep the old key to account for older Jetpack versions. There might be other things this affects that I am not sure of, such as Activity Log (maybe)?

I'd say it's not worth the effort to rename, and instead use our old friend: inline comments.

Reply:
This is the test file, I'm adding the comment to `sync/class.jetpack-sync-defaults.php` instead.

Reply:
4f027dbe8